docs/data_sources/data_catalog.rst

```diff
@@ -0,0 +1,126 @@
+.. _data-catalog:
+
+===============================================================================
+Data Catalog
+===============================================================================
+
+.. contents::
+
+-------------------------------------------------------------------------------
+Available Data
+-------------------------------------------------------------------------------
+
+.. _data-epaipm:
+
+EPA IPM
```

Comment:
we are not really supporting IPM right now so I think we can remove this.

Reply:
Should I also remove all references to the IPM in the documentation?

Reply:
done